src/WPUpdatePhp.php

```diff
@@ -27,7 +27,21 @@ public function __construct( $minimum_version, $recommended_version = null ) {
 	 * @param $name string Name of the plugin to be used in admin notices
 	 */
 	public function set_plugin_name( $name ) {
-		$this->plugin_name = $name;
+		if ( empty( $this->plugin_name ) ) {
```

Comment:
I think this if-statement needs to be the other way around? Either way, do you think this is a good case, as the parameter is required and if you purposefully try to break it by passing `NULL` or anything like that, I'm fine with it displaying weird in the admin notice. Don't you agree?